lib/tasks/generate_test_trainee_csv.rake

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+def generate_row_data
```

Comment:
maybe create a factory for these or using the existing trainee factory to helper generate it

```ruby

keys = ["first_names", "last_name", "date_of_birth", "sex", "email", "training_route", "itt_start_date", "itt_end_date", "course_subject_one", "study_mode", "hesa_id", "provider_trainee_id"]
trainee = FactoryBot.build(:trainee)
trainee_attributes = trainee.attributes

new_hash = trainee_attributes.slice(*keys)

```